cmd/eib/main.go

```diff
@@ -1,7 +1,74 @@
 package main
 
-import "fmt"
+import (
+	"flag"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/suse-edge/edge-image-builder/pkg/config"
+)
+
+const (
+	argConfigFile = "config"
+	argVerbose    = "verbose"
+)
+
+func init() {
+	log.SetFormatter(&log.TextFormatter{
+		FullTimestamp:    true,
+		QuoteEmptyFields: true,
+	})
+	log.SetOutput(os.Stdout)
+}
+
+func processArgs() (imageConfig *config.ImageConfig) {
```

Comment:
`processArgs`, as the name suggests, should not return anything. It's better to inline the parsing in the main function.

Bonus tip: The return values should not be named unless you specifically use the variables for some type of modification.

Reply:
You read my mind, I was gonna specifically ask about the named return values.

Reply:
The first part I disagree with though. IMO, `processArgs` should do everything related to the arguments to convert them from the CLI call into the pieces that will be handed to the image build process. I'm open to another name if the term `process` feels misleading (I intentionally chose that over simply `parse`), but I like the idea of a single entry point that does all of the messing around with arguments.

I'm going to leave it as is for now and see how it evolves when/if we add new flags.